app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -266,13 +266,21 @@ def after_create_response
         end
       end
 
+      def update_referer
+        if params['referer_anchor'].present?
```

Comment:
Two things here:

1. Not sure you need the `#present?` here. If the key isn't present, I believe `params[]` will return `nil`.
2. What would you think of replacing the `'referer_anchor'` string in lines 270 and 271 with a symbol?